client/contracts/bindings/consensus/build/DomainResolver.abi

```diff
@@ -0,0 +1 @@
+[{"constant":true,"inputs":[{"name":"node","type":"bytes32"}],"name":"addr","outputs":[{"name":"","type":"address"}],"payable":false,"stateMutability":"view","type":"function"}]
```

Comment:
abi/bins should be removed

Reply:
There's a separate issue for that. There are many binaries in the codebase that need to be removed.